src/Sulu/Component/Content/Types/Rlp/Strategy/RlpStrategy.php

```diff
@@ -149,6 +149,10 @@ public function save(ResourceSegmentBehavior $document, $userId)
 
         $this->mapper->save($document);
 
+        if ($treeValue === '/') {
```

Comment:
this check should be replaced with `$document instanceof Homedocument` normal pages should not have '/'. in an ideal world there should be an exception if a page has '/' as resource-locator.